### ISSUE TITLE
Is this api working?

### ISSUE THREAD

**Issue by devWaleed** [Biboxcom/api_reference#6]
I am trying to fetch all assets/balances from my account but I get 3011 error. I have tried everything but its not working. Can someone confirm if this API is still working for the following command:

cmd: transfer/assets
body{ select : 1 }

**Comment by MaxPovver** [comment]
Yes, it works, you must pass commands as array even if you have only one command.
I can show you working PHP example

**Comment by devWaleed** [comment]
Yes please, I am working on C# thought but i would like to see a PHP custom example.

**Comment by MaxPovver** [comment]
@devWaleed check out this repo https://github.com/MaxPovver/biboxDemo

**Comment by devWaleed** [comment]
@MaxPovver Thanks. I'll give it a try.

**Comment by yuandong618** [comment]
@devWaleed   I am working on C# too
